flow/util/template_filters.py

```diff
@@ -11,12 +11,12 @@
 
 
 def identical(iterable):
-    """Check that all elements of an iterator are identical"""
+    """Check that all elements of an iterator are identical."""
```

Comment:
Technically this should be an iterable.
```suggestion
    """Check that all elements of an iterable are identical."""
```